packages/cli/generators/app/templates/src/application.ts.template

```diff
@@ -1,13 +1,22 @@
-import {ApplicationConfig} from '@loopback/core';
+import {Application, ApplicationConfig, BootOptions} from '@loopback/core';
 import {RestApplication, RestServer} from '@loopback/rest';
-import {PingController} from './controllers/ping.controller';
 import {MySequence} from './sequence';
 
-export class <%= project.applicationName %> extends RestApplication {
+/* tslint:disable:no-unused-variable */
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '@loopback/boot';
+/* tslint:enable:no-unused-variable */
+
+export class <%= project.applicationName %> extends BootMixin(RestApplication) {
   constructor(options?: ApplicationConfig) {
     super(options);
-    this.sequence(MySequence);
-    this.setupControllers();
+    this.projectRoot = __dirname;
+    // Customize @loopback/boot Booter Conventions here
+    this.bootOptions = {
+      controllers: {
+        // Customize ControllerBooter Conventiones here
```

Comment:
Could you please spell out all default configuration options here? It will make it easier for users to immediately see which options are available for tweaking.